### PR TITLE
CB-6084 add verifyPermissions flag to credential

### PR DESCRIPTION
**Pull request by sodre90**

### CODE REVIEW

File: cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsCredentialConnector.java [hunk 1]
```diff
@@ -135,11 +98,53 @@ private CloudCredentialStatus verifyAccessKeySecretKeyIsAssumable(CloudCredentia
...
+            try {
+                awsCredentialVerifier.validateAws(awsCredential);
+            } catch (AwsPermissionMissingException e) {
+                return new CloudCredentialStatus(cloudCredential, CredentialStatus.PERMISSIONS_MISSING, new Exception(e.getMessage()), e.getMessage());
+            }
         }
         return new CloudCredentialStatus(cloudCredential, CredentialStatus.VERIFIED);
     }
+
+    @Override
```
Comment (doktoric) [inline cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsCredentialConnector.java]: as I see this method did not changed but on an other. Can we put back? This is just noise in the PR

File: environment/src/main/resources/schema/app/20200316133300_CB-6084_add_verify_to_credential.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+-- // CB-6084 add verify column to credential
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE credential ADD COLUMN IF NOT EXISTS verifyPermissions boolean default true;
```
Comment (doktoric) [inline environment/src/main/resources/schema/app/20200316133300_CB-6084_add_verify_to_credential.sql]: not null ?

Comment (sodre90) [reply environment/src/main/resources/schema/app/20200316133300_CB-6084_add_verify_to_credential.sql]: good catch, thanks

File: cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsCredentialConnector.java (hunk 1 above)
Comment (sodre90) [reply cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsCredentialConnector.java]: yes, idea rearranged the code and I didn't recognize, thanks

**Pull request merged**